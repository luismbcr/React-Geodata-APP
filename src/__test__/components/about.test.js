import React from 'react';
import { shallow } from 'enzyme';
import About from '../../components/about';

test('Should render About Component', () => {
  const wrapper = shallow(<About />);
  expect(wrapper).toMatchSnapshot();
});
