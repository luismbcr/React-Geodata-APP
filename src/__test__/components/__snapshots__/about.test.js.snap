// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render About Component 1`] = `
<div>
  <Header
    as="h2"
  >
    About this project
  </Header>
  <p>
    This app consumes GeoNames's geographical database  to show information about countries and cities, Countryflag to get flags and HolidayAPI to get holiday information.
  </p>
  <Label
    as="a"
    content="Github"
    href="https://github.com/luismbcr/React-Geodata-APP"
    icon="github"
    target="_blank"
  />
</div>
`;
